test/rosdep_repo_check/config.yaml

```
@@ -54,6 +54,7 @@ supported_versions:
   alpine:
   - edge
   debian:
+  - bullseye
   - buster
```

The 'latest' release of a platform should appear last. It's the one that suggestions are made for.
```suggestion
  - buster
  - bullseye
```

---

Thanks. I was following the rosdep requirements/convention that definitions be in lexical order but I like release order better so I'm happy to accept this change and it's good to know that the suggested platform is the last one.